src/field_info.rs

```diff
@@ -231,6 +232,11 @@ impl SetterSettings {
                         self.doc = Some(*assign.right);
                         Ok(())
                     }
+                    "transform" => {
```

Comment:
`transform` cancels `strip_option`, so I think we should make it illegal to use put both on the same field. This can be tricky though (requires a change to the `handle_fields` macro) so I might do it myself after merging.

Reply:
The automatic cancellation is intended for the `field`defaults` one. Explicitly specifying both in the field attribute would be unreasonable, but I found it not so easy to distinguish, so I left it alone for the while.